<launch>
    <node pkg="ssf_updates" type="visionpose_sensor" name="ekf_odometer"   clear_params="true" output="screen">
			<remap from="ekf_odometer/imu_state_input" to="/imu/imu0" />
			<remap from="ekf_odometer/mag_state_input" to="/imu/mag0" />
         	<remap from="ekf_odometer/visionpose_measurement" to="/stereo_odometer/pose_iw_" />

	    	<rosparam file="$(find ssf_updates)/visionpose_sensor_fix.yaml"/>
    </node>
</launch>
